src/models/apiManagement.ts

```diff
@@ -16,6 +16,10 @@ export interface ApiManagementConfig {
   jwtValidate?: ApiJwtValidatePolicy;
   /** The API's IP Filter policy */
   ipFilter?: ApiIpFilterPolicy;
+  /** The API's IP Filter policies */
+  ipFilters?: ApiIpFilterPolicy[];
```

Comment:
what's the difference between `ipFilter` & `ipFilters`?

Reply:
Described above in previous comment.